src/types/tags.ts

```diff
@@ -5,12 +5,22 @@ export enum TagTypes {
   Oskused = 'Oskused',
 }
 
+export interface TagFields extends Object {
```

Comment:
**[blocker]**

This looks like a duplicate of the "Tag" type. What is the difference between them and should there be any ? They are both talking about the type of the same object